models/public/single-human-pose-estimation-0001/model.yml

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+
+  The pose may contain up to 17 keypoints: ears, eyes, nose,
+  shoulders, elbows, wrists, hips, knees and ankles.
+
+  The model input is a blob that consists of one images of "1x3x384x288" in
+  RGB order.
+  The model outputs one blob: heatmaps ( shape of output blob "1x17x48x36")
+task_type: human_pose_estimation
+files:
+  - name: single-human-pose-estimation-0001.tar.gz
+    sha256: 182c51075502643ada303a5f3a081b6adf943b711e5eddf80635582369588d34
+    size: 151979155
+    source: https://download.01.org/opencv/openvino_training_extensions/models/human_pose_estimation/single-human-pose-estimation-0001.tar.gz
+
+postprocessing:
+  - $type: unpack_archive
+    format: gztar
+    file: single-human-pose-estimation-0001.tar.gz
+framework: pytorch
+conversion_to_onnx_args:
+  - --model-path=$conv_dir/single-human-pose-estimation-0001/
+  - --model-name=SinglePersonPoseEstimationWithMobileNet
+  - --import-module=models
+  - --weights=$conv_dir/single-human-pose-estimation-0001/single-human-pose-estimation-0001.pth
+  - --input-shape=1,3,384,288
+  - --input-names=data
+  - --output-names=heatmaps0,heatmaps1,heatmaps2,heatmaps3,heatmaps4,heatmaps
```

Comment:
This seems odd. The description claims that the original model only has 1 output, so why are 6 outputs listed here?

Reply:
@IRDonch   Yes, the model has 6 outputs. This is necessary in order to calculate the loss in during training. But in the future,  for obtaining the results of prediction and postprocessing , the last output "heatmaps" is used. IR model has one output.

Reply:
OK, but then the documentation for the original model outputs should explain that.